### PR TITLE
Support banco bolivariano

### PR DESCRIPTION
**Pull request by 0xjams**
This PR adds support for a bank's debit/credit format.

Changes in this pull request:

-
-
-

@JC5

### CODE REVIEW

File: .github/workflows/sonarcloud.yml
Comment (JC5) [inline .github/workflows/sonarcloud.yml:1]: Could you remove this change from the PR please? Thanks!

**Comment by 0xjams** [comment]
I didn't explicitly modify that file. Weird! Reverting it right now :)

**Comment by 0xjams** [comment]
So I think I know what happened, I forked the main branch, and in that case, the only difference is the file I modified:

Could you check the pull request I accidentally sent to the main branch? It only affected the right file.
https://github.com/firefly-iii/data-importer/pull/637

However, now that I sent the same pull request to the develop branch, it seems the other file is affected too.  Meaning (I think) that there's a slight difference between that file in those branches.

I could start from scratch using the develop version as the one I fork.  Just let me know how you want to proceed.

**Comment by 0xjams** [comment]
I opened a new one
https://github.com/firefly-iii/data-importer/pull/639

**Pull request closed without merging**